src/Microsoft.Framework.CodeGeneration.Common/CommonUtilities.cs

```diff
@@ -6,14 +6,12 @@
 using System.Reflection;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Emit;
-using Microsoft.Framework.Runtime;
 
 namespace Microsoft.Framework.CodeGeneration
 {
     internal static class CommonUtilities
     {
         public static CompilationResult GetAssemblyFromCompilation(
-            IAssemblyLoadContext loader,
```

Comment:
Why don't you just implement one on net45 and pass it in instead of using reflection?